src/makeCalculator.js

```diff
@@ -35,7 +35,53 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(callback, number) {
+      this.result = callback(this.result, number);
+
+      return this;
+    },
+
+    add(res, value) {
+      let operationResult = res;
+
+      operationResult += value;
+
+      return operationResult;
+    },
```

Comment:
add(res, value) {
       return res + value;
    },

Reply:
Thank you for your advice! 👍